site/docs/install-os-x.md

```diff
@@ -141,3 +141,11 @@ following command:
 ```shell
 $ brew upgrade bazel
 ```
+
+To install older bazel versions
+
+```
+brew tap bazelbuild/tap
+brew extract bazel bazelbuild/tap --version X.Y.Z
```

Comment:
> I'm not sure what previous versions are available via brew.

[docs say](https://docs.brew.sh/Manpage#extract---version---force-formula-tap)

> Look through repository history to find the most recent version of formula and create a copy in tap

I was able to extract 4.0.0 and 0.29.0. I think what versions are available depend on the history.

```
brew extract bazel bazelbuild/tap --version 0.29.0
==> Searching repository history
==> Writing formula for bazel from revision 16d0135 to:
/opt/homebrew/Library/Taps/bazelbuild/homebrew-tap/Formula/bazel@0.29.0.rb
```